src/sql/workbench/contrib/notebook/browser/calloutDialog/media/calloutDialog.css

```diff
@@ -49,7 +42,6 @@
 	transform: rotate(135deg);
 }
 .callout-arrow.from-left:before {
```

Comment:
Would these be applicable to ALL callout dialogs or just the notebook specific ones? If it's all then they should really go in modal.css

Reply:
The calloutDialog style is a concept that's current in play in notebook but certainly not limited to notebook. I would elect to keep these styles in calloutDialog.css. Modal and flyout type do not use this style. I did what I could to make these generic enough so that, when calling the calloutDialog, one can specify the dialog position with respect to the triggering element.

Reply:
You can put them in the css but still keep the rule selectors as they are and it would be the same thing. All the styles get bundled into a huge single CSS anyways so it's not like it'd really be changing anything.

The point of moving it would be to keep the styles next to the thing that actually uses it. So since these apply to all callout dialogs they should go in modal.css (or if you prefer you can move this file over there and then also import this css in modal.ts)